index.html

```diff
@@ -182,6 +182,25 @@ <h2> About <br> Myself</h2>
             </ul>
           </div>
           <hr>
+
+          <section class="section-form">
+            <div class="form-div">
+             <h3>Contact me</h3>
+             <p>If you have an application you are 
+               interested in developing,
+               a feature that you need built or a project that 
+               needs coding. I’d love to help with it </p>
+            </div>
+            <form action="/my-handling-form-page" method="post">
+               <ul>
+              <li><input type="text" id="name" name="user_name" placeholder="Okpah Igbo" maxlength="30" required></li>
+              <li><input type="email" id="mail" name="user_email" placeholder="davidokpah396@gmail.com" required></li>
```

Comment:
- [OPTIONAL] Good job adding `placeholder` attribute to each input field but I think it would be better if you change it for a more descriptive text like 'Write your name here' for name and 'Write your email here' for email :+1:.